src/main/java/emissary/core/IBaseDataObjectDiffHelper.java

```diff
@@ -157,8 +158,10 @@ public static void diff(final SeekableByteChannelFactory sbcf1, final SeekableBy
                     InputStream is1 = Channels.newInputStream(sbc1);
                     InputStream is2 = Channels.newInputStream(sbc2)) {
                 if (!IOUtils.contentEquals(is1, is2)) {
-                    differences.add(String.format("%s not equal. 1.cs=%s 2.cs=%s",
-                            identifier, sbc1.size(), sbc2.size()));
+                    differences.add(String.format("%s not equal. 1.is=%s 2.is=%s",
```

Comment:
We definitely don't want to spit out a string representation of the entire payload; that could be well over a GB for each object being compared.

Reply:
That makes sense, I hadn't considered that.

Reply:
Would it make sense to have it display a string representation of the first `x` characters? Where `x` is set to 64 for ease of debugging

Reply:
We could consider making that an option, but it would only be helpful if 1) the payload is text-like and decodes well using UTF_8  2) the difference occurs within the first few bytes.